JUnit4Testing/src/test/java/StringUtilsTest.java

```diff
@@ -0,0 +1,68 @@
+import org.example.StringUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+public class StringUtilsTest
+{
+    StringUtils stringUtils;
+    @Before
+    public void setup()
+    {
+        stringUtils = new StringUtils();
+    }
+
+    // Testing:- reverse method with non-empty string.
+    @Test
+    public void testReversee_test1()
```

Comment:
_test1 --> wrong 

_should_reverse_given_string , we need to mention what are we testing